encord/orm/dataset.py

```diff
@@ -28,8 +28,9 @@
 
 DATETIME_STRING_FORMAT = "%Y-%m-%d %H:%M:%S"
 
+
 class DatasetUserRole(IntEnum):
-    ADMIN = 0,
+    ADMIN = (0,)
```

Comment:
Why has this got brackets around it but the 1 below doesn't?

Reply:
I assume because of the added `,`. I will remove it. That's a good catch